example/lib/pages/latlng_to_screen_point.dart

```diff
@@ -0,0 +1,86 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map/plugin_api.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../widgets/drawer.dart';
+
+class LatLngScreenPointTestPage extends StatefulWidget {
+  static const String route = 'latlng_screen_point_test_page';
+
+  const LatLngScreenPointTestPage({Key? key}) : super(key: key);
+
+  @override
+  State createState() {
+    return _LatLngScreenPointTestPageState();
+  }
+}
+
+class _LatLngScreenPointTestPageState extends State<LatLngScreenPointTestPage> {
+  late final MapController mapController;
+  late final StreamSubscription<MapEvent> subscription;
+
+  CustomPoint textPos = const CustomPoint(10.0, 10.0);
+
+  @override
+  void initState() {
+    super.initState();
+    mapController = MapController();
+    subscription = mapController.mapEventStream.listen(onMapEvent);
+  }
+
+  @override
+  void dispose() {
+    subscription.cancel();
+
+    super.dispose();
+  }
+
+  void onMapEvent(MapEvent mapEvent) {
+    if (mapEvent is! MapEventMove && mapEvent is! MapEventRotate) {
+      // do not flood console with move and rotate events
+      debugPrint(mapEvent.toString());
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Test out Interactive flags!')),
```

Comment:
This needs correcting:

```suggestion
        appBar: AppBar(title: const Text('LatLng To Screen Point')),
```